packages/gettext.rb

```diff
@@ -23,15 +23,19 @@ class Gettext < Package
      x86_64: '4ada2428329f3fc4098b4fab8cc81d816028a7ac283c359c86fdc1d569a52af3'
   })
 
+  depends_on 'acl' # R
+  depends_on 'gcc' # R
```

Comment:
Not a build dependency?

Reply:
Might be a build dependency, but also a runtime dependency.